lib/activity.js

```diff
@@ -30,7 +30,7 @@ export const createActivityStream = source => {
 const activityLogLineToObject = line => {
   const { date, text } = parseLog(line)
   const { type, message, source, id } = JSON.parse(text)
-  return { timestamp: date.toJSON(), type, source, message, id }
+  return { timestamp: date.toISOString(), type, source, message, id }
```

Comment:
I believe `Date.prototype.toJSON()` is using `toISOString()` under the hood, so this change is not necessary. 

Quoting from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toJSON:
> [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) instances refer to a specific point in time. toJSON() calls the object's [toISOString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) method, which returns a string representing the [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object's value. This method is generally intended to, by default, usefully serialize [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) objects during [JSON](https://developer.mozilla.org/en-US/docs/Glossary/JSON) serialization, which can then be deserialized using the [Date() constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) or [Date.parse()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse) as the reviver of [JSON.parse()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse).

In fact, why do we have to convert the date from `Date` to `String` here? I would expect the conversion to happen in the place that's converting the activity log line object to a string, as that's the place where we need to make the decision.

Reply:
I removed the conversion and changed it to be `timestamp` everywhere. I initially didn't do it because to me a property called `date` would be of type `Date`, any other type (like `String` or `Number` would be called something else. I realized now however that called a `Date` a `timestamp` makes sense. In my code before, a timestamp I think would always refer to a unix epoch.

Reply:
Excellent! I find the new code much easier to comprehend 👍🏻